crates/aggchain-proof-builder/src/lib.rs

```diff
@@ -279,6 +280,28 @@ impl<ContractsClient> AggchainProofBuilder<ContractsClient> {
             })
             .collect();
 
+        let removed_gers: Vec<Digest> = request
+            .aggchain_proof_inputs
+            .removed_gers
+            .into_iter()
+            .filter(|removed_ger| new_blocks_range.contains(&removed_ger.block_number))
+            .collect::<Vec<_>>()
+            .into_iter()
+            .sorted()
+            .map(|removed_ger| removed_ger.global_exit_root)
+            .collect();
```

Comment:
```suggestion
        let mut removed_gers = request
            .aggchain_proof_inputs
            .removed_gers
            .into_iter()
            .filter(|removed_ger| new_blocks_range.contains(&removed_ger.block_number))
            .collect::<Vec<_>>();
        removed_gers.sort();
        let removed_gers: Vec<Digest> = removed_gers
            .into_iter()
            .map(|removed_ger| removed_ger.global_exit_root)
            .collect();
```

That way, we can eliminate the need for `itertools` altogether.